app/src/renderer/components/staking/PageNominate.vue

```diff
@@ -122,6 +124,7 @@ import FormGroup from '../common/NiFormGroup'
 import FormMsg from '../common/NiFormMsg'
 import FormStruct from '../common/NiFormStruct'
```

Comment:
don't need the .. anymore

Reply:
Yep, will work on a PR to remove all of those relative links (soon).